incubator/openstack-ironic-standalone/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+name: openstack-ironic-standalone
+version: 0.0.1
```

Comment:
If I'm not wrong, in `Semantic Versioning` (and therfore in `helm/charts`) the first version needs to be `0.1.0`

Reply:
Thank you - I changed that to `0.1.0`